packages/ui/src/kit/VLinkCollapse/VLinkCollapse.theme-default.scss

```diff
@@ -0,0 +1,18 @@
+.v-link-collapse.theme-default {
+  display: flex;
+  font-size: 14px;
+  text-decoration: none;
+  align-items: center;
+  justify-content: center;
+  color: var(--endpass-ui-color-primary-7);
+  cursor: pointer;
+}
+
+.v-link-collapse.theme-default .v-link-collapse-chevron {
+  margin-left: 10px;
+  transition: transform .3s;
```

Comment:
Try to make animation faster. Because text changes immediately, and chevron looks like it is late. Looks artificial.

Reply:
i was selected `0.3s` because in most other animation cases we are using same value